Tasks/DownloadGithubReleasesV0/githubreleases.handlebars.txt

```diff
@@ -0,0 +1,13 @@
+[
+{{#this}}
+  {
+    "path": "{{this.name}}",
+    "lastModified": "{{this.updated_at}}",
+    "fileLength" : "{{this.size}}",
```

Comment:
Does this need contentType changes?

Reply:
with Authorization Header fix, now its working fine with using **browser_download_url**, so we don't need **contentType**